### ISSUE TITLE
[Question] Is there some sort of caching happening? If so, how to clear it?

### ISSUE THREAD

**Issue by abeforgit** [lerna/lerna-changelog#915]
So I've noticed recently that some prs are not picked up even though they are properly labeled (we use the default labels).
What they have in common is that they're all pr's that were merged very quickly by myself (usually in the frantic hours of late-night debugging)
Is there some sort of debounce/caching/rate limiting happening? (I have the github token set up as per the docs)

I understand it's not really a typical usecase, but sometimes the PR flow is too much overhead but you still want to keep nice changelogs. So if this is expected behavior that's totally fair. But maybe it would be good to document it?

**Comment by Turbo87** [comment]
yes, caching is used, but it should be using the regular HTTP caching semantics, so it seems somewhat unlikely to get into a situation where a stale cache exists 🤔

**Comment by abeforgit** [comment]
Interesting. It may be on github's end, I'll monitor and see if I can get a useful repro

**Comment by abeforgit** [comment]
I haven't encountered this any more, and you did answer the original question, so I think you can close this issue. Thank you for your time.
